build/android/bytecode/java/org/brave/bytecode/BraveManageSyncSettingsClassAdapter.java

```diff
@@ -16,6 +16,9 @@
     BraveManageSyncSettingsClassAdapter(ClassVisitor visitor) {
         super(visitor);
 
+        deleteField(sBraveManageSyncSettingsClassName, "mTurnOffSync");
+        makeProtectedField(sManageSyncSettingsClassName, "mTurnOffSync");
```

Comment:
Could you also add unit test to BytecodeTest.java to check that this variable still exist in parent class?

Reply:
Updated the test at https://github.com/brave/brave-core/pull/9799/commits/1da551807122e4ec6c5b19fea15bfbb7ad113ffd . Thanks for the help.